src/Qowaiv.Diagnostics.Contracts/Qowaiv.Diagnostics.Contracts.csproj

```diff
@@ -12,8 +12,11 @@
     <Version>2.0.5</Version>
```

Comment:
Does version need to be updated as well?

Reply:
Oops